services/algolia-sync/src/components/algolia/site-sync.js

```diff
@@ -0,0 +1,81 @@
+const gql = require('graphql-tag');
+const algoliasearch = require('algoliasearch');
+const cliProgress = require('cli-progress');
+const apollo = require('../graphql/query');
+const helpers = require('./helpers');
+const { ALGOLIA_APPID, ALGOLIA_API_KEY } = require('../../env');
+
+const client = algoliasearch(ALGOLIA_APPID, ALGOLIA_API_KEY);
+
+const bar = new cliProgress.SingleBar({}, cliProgress.Presets.shades_classic);
+
+const query = gql`
+  query websiteScheduledContent($input: AllPublishedContentQueryInput!) {
+    allPublishedContent(input: $input){
+      pageInfo {
+        hasNextPage
+      }
+      edges {
+        node {
+          id
+          type
+          created
+          published
+          updated
+          status
+          websiteSchedules { section { hierarchy { fullName } }}
+          primaryImage { name filePath fileName src isLogo }
+          primarySite{ host }
+          name
+          teaser
+        }
+      }
+    }
+  }
+`;
+
+const total = () => (gql`
+  query websiteScheduledContent($input: AllPublishedContentQueryInput!) {
+    allPublishedContent(input: $input){
+      totalCount
+    }
+  }
+`);
+
+const upsertToIndex = async (tenant, limit, skip, totalContent) => {
+  bar.update(skip);
+
+  const input = {
+    pagination: {
+      limit,
+      skip,
+    },
+  };
+
+  const data = await apollo.queryFromBase(query, tenant, input);
+  const nodes = data.allPublishedContent.edges.map(content => content.node);
+  const algoliaData = await helpers.buildObj(nodes, tenant);
+  await client.multipleBatch(algoliaData, (err) => { if (err) throw err; });
```

Comment:
The `multipleBatch` function returns a promise (which you're awaiting). As such, there's no need to include the `(err) => {}` callback.